### PR TITLE
Add support for arm64-v8a on GoogleVR, Oculus and SVR

### PR DESCRIPTION
**Pull request by MortimerGoro**

### CODE REVIEW

**Comment by bluemarvin** [comment]
The title confused me, I didn't think Wave had 64 bit libs. Should be GVR instead of Wave 🙂

File: app/build.gradle [hunk 1]
```diff
@@ -168,10 +175,16 @@ android {
...
                 'oculusvrArmRelease',
+                'oculusvrArm64Debug',
+                'oculusvrArm64Release',
                 'svrArmDebug',
                 'svrArmRelease',
+                'svrArm64Debug',
+                'svrArm64Release',
                 'wavevrArmDebug',
                 'wavevrArmRelease',
                 'noapiArmDebug',
```
Comment (bluemarvin) [inline app/build.gradle]: Can you add noapiArm64[Debug,Release] also?

Comment (MortimerGoro) [reply app/build.gradle]: Done

**Comment by MortimerGoro** [comment]
I also updated the GeckoView version. Today's 64bit nighlty showed much more performance on https://threejs.org/examples/webvr_ballshooter.html

> The title confused me, I didn't think Wave had 64 bit libs. Should be GVR instead of Wave

Yep, the mistake was only on the commit description :sweat_smile:

**Pull request merged**